src/theme/Footer/index.tsx

```diff
@@ -226,7 +229,7 @@ function Footer(): JSX.Element | null {
           </ul>
         </div>
         <div className={styles.minaFooter_legals__info}>
-          ©2022 Mina. Started by O(1) Labs.
+          ©2023 Mina. Started by O(1) Labs.
```

Comment:
```suggestion
          ©2022 Mina. Started by O(1) Labs.
```

Reply:
let's not update the copyright each year, per best practices 
![image](https://github.com/o1-labs/docs2/assets/10067215/13960221-3d2d-4928-9698-d8a72d31af68)
 
 Teach me if you know otherwise
 
 What does it mean "Started by O(1) Labs"? Do we mean that Mina Protocol was incubated by O(1) Labs?  
 
 I wonder who originally wrote this footer text? Let's get Maki's eyes on it

Reply:
On the copyright, wouldn't that mean we've to take it back to the year the o1 was truly launched then?
Personally, i think it's okay to update it so far we're making improvements on the website but open to other suggestions.

As for the "Started by O(1) Labs" I just decided to leave what existed on the current doc page, but yeah Maki might provide better opinions on that.

Reply:
I did some checking around, and your instinct is on target. I stand corrected, let's keep your suggestion and update our copyright, thank you @LucidSamuel 

GitHub docs https://docs.github.com/en
![image](https://github.com/o1-labs/docs2/assets/10067215/d9673a9b-42a6-43ed-8440-648c8bc2df4a)

Frame Platform docs  https://docs.fra.me/
![image](https://github.com/o1-labs/docs2/assets/10067215/071692aa-daf4-41e8-b561-67872e544fc3)

Microsoft Visual Studio Code (same throughout all MS docs) https://learn.microsoft.com/en-us/visualstudio/javascript/?view=vs-2022 
![image](https://github.com/o1-labs/docs2/assets/10067215/7a9d897d-1730-442e-822e-d6688e0b1fb3)

DataStax (I'm an alumni) https://docs.datastax.com/en/home/docs/index.html
![image](https://github.com/o1-labs/docs2/assets/10067215/68aa49fa-85f6-429f-8913-86b2a38e41ed)

Substrate https://docs.substrate.io/tutorials/
![image](https://github.com/o1-labs/docs2/assets/10067215/0bfb9a2e-59de-4222-ba33-19b42bf58e54)

Linode (a range of copyright years) https://www.linode.com/docs/guides/
![image](https://github.com/o1-labs/docs2/assets/10067215/9badae40-8f90-4b40-b197-858a65f8328d)